File: src/test/java/io/github/vivek/quotes/QuoteBuilderTest.java
package io.github.vivek.quotes;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class QuoteBuilderTest {
    @Test
    public void testBuildQuotes() throws Exception {
        assertNotNull(QuotesListBuilder.quotes);
        assertTrue(QuotesListBuilder.quotes.size() != 0);
    }

}

File: src/test/kotlin/io/github/vivek/quotes/QuoteBuilderTest.kt
package io.github.vivek.quotes

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.*

class QuoteBuilderTest {
    @Test
    @Throws(Exception::class)
    fun testBuildQuotes() {
        assertNotNull(QuotesListBuilder.quotes)
        assertTrue(QuotesListBuilder.quotes.isNotEmpty())
    }
}